### PR TITLE
Update SC wasm files

### PR DESCRIPTION
**Pull request by sydhds**
* [ ] document all added functions
* [ ] try in sandbox /simulation/labnet
  * [ ] if part of node-launch, checked using the `resync_check` flag
* [ ] unit tests on the added/changed features
  * [ ] make tests compile
  * [ ] make tests pass 
* [ ] add logs allowing easy debugging in case the changes caused problems
* [ ] if the API has changed, update the API specification

### CODE REVIEW

File: massa-execution-worker/src/interface_impl.rs [hunk 1]
```diff
@@ -690,11 +690,11 @@ impl Interface for InterfaceImpl {
...
     ///
     /// [DeprecatedByNewRuntime] Replaced by `get_op_keys_wasmv1`
-    fn get_op_keys(&self) -> Result<Vec<Vec<u8>>> {
+    fn get_op_keys(&self, prefix_opt: Option<&[u8]>) -> Result<Vec<Vec<u8>>> {
         // TODO return BTreeSet<Vec<u8>>
 
-        // TODO add prefix
-        let prefix: &[u8] = &[];
+        let prefix: &[u8] = prefix_opt.unwrap_or_default();
+        // let prefix: &[u8] = Default::default();
```
Comment (damip) [inline massa-execution-worker/src/interface_impl.rs:697]: this kind of comments are only causing confusion to new readers and should always be removed

File: Cargo.toml [hunk 2]
```diff
@@ -99,7 +99,8 @@ massa_wallet = { path = "./massa-wallet" }
 
 # Massa projects dependencies
 massa-proto-rs = {git = "https://github.com/massalabs/massa-proto-rs", "rev" = "c7a29753d7d31864779db01e8260a8bf0748e10c"}
-massa-sc-runtime = {git = "https://github.com/massalabs/massa-sc-runtime", "branch" = "main"}
+# massa-sc-runtime = {git = "https://github.com/massalabs/massa-sc-runtime", "branch" = "feature/get_op_keys_prefix"}
```
Comment (damip) [inline Cargo.toml:102]: This kind of comments are only causing confusion to new readers and should always be removed. Git history is there if they need to see the modifications

File: massa-models/src/address.rs [hunk 3]
```diff
@@ -681,16 +681,16 @@ mod test {
         // println!("sc_addr_1: {}", sc_addr_1);
 
         // let v1 = "AU12M3AQqs7JH7mSe1UZyEA5NQ7nGQHXaqqxe1TGEpkimcRhsQ4eF";
-        let v2 = "AU4cJWyjpBetGwaRqFDXyrHiQuGB3QKrwjzGiGSzQPGeAARB9AY4";
-        let addr = Address::from_str(v2).unwrap();
+        // let v2 = "AU4cJWyjpBetGwaRqFDXyrHiQuGB3QKrwjzGiGSzQPGeAARB9AY4";
```
Comment (damip) [inline massa-models/src/address.rs:684]: This kind of comments are only causing confusion to new readers and should always be removed. Git history is there if they need to see the modifications

File: massa-execution-worker/src/interface_impl.rs [hunk 4]
```diff
@@ -690,11 +690,11 @@ impl Interface for InterfaceImpl {
     /// A list of keys (keys are byte arrays)
     ///
     /// [DeprecatedByNewRuntime] Replaced by `get_op_keys_wasmv1`
-    fn get_op_keys(&self) -> Result<Vec<Vec<u8>>> {
+    fn get_op_keys(&self, prefix_opt: Option<&[u8]>) -> Result<Vec<Vec<u8>>> {
         // TODO return BTreeSet<Vec<u8>>
```
Comment (damip) [inline massa-execution-worker/src/interface_impl.rs:694]: this comment can be removed

File: massa-execution-worker/src/interface_impl.rs (hunk 1 above)
Comment (sydhds) [reply massa-execution-worker/src/interface_impl.rs]: Done

File: Cargo.toml (hunk 2 above)
Comment (sydhds) [reply Cargo.toml]: Done

File: massa-models/src/address.rs (hunk 3 above)
Comment (sydhds) [reply massa-models/src/address.rs]: Done

File: massa-execution-worker/src/interface_impl.rs (hunk 4 above)
Comment (sydhds) [reply massa-execution-worker/src/interface_impl.rs]: Done

**Review by damip — APPROVED** [review]

File: Cargo.toml [hunk 5]
```diff
@@ -99,7 +99,7 @@ massa_wallet = { path = "./massa-wallet" }
 
 # Massa projects dependencies
 massa-proto-rs = {git = "https://github.com/massalabs/massa-proto-rs", "rev" = "c7a29753d7d31864779db01e8260a8bf0748e10c"}
-massa-sc-runtime = {git = "https://github.com/massalabs/massa-sc-runtime", "branch" = "main"}
+massa-sc-runtime = {git = "https://github.com/massalabs/massa-sc-runtime", "branch" = "feature/get_op_keys_prefix"}
```
Comment (damip) [inline Cargo.toml:102]: now that it is merged on the runtime side, the dependency should be updated and run "cargo update"

**Pull request merged**